web/modules/custom/dpl_event/src/Services/EventRestMapper.php

```diff
@@ -63,6 +63,7 @@ public function getResponse(EventInstance $event_instance): EventsGET200Response
       'branches' => $this->getBranches(),
       'address' => $this->getAddress(),
       'tags' => $this->getTags(),
+      'partners' => $this->getMultiValue('event_partners'),
```

Comment:
Consider what happens with this value if there are no partners.

The spec requires an optional array of strings.`getMultiValue()` returns `null` or an array of strings. Will the response leave out the `partners` property if the value is `null` or automatically convert it to an empty array?

Personally I would prefer `getMultiValue()` to always return an array of strings. The value might then be empty.

Reply:
Fair point. I've updated it so it'll always return an array, even if it is empty.